bundles/org.openhab.binding.telegram/src/main/java/org/openhab/binding/telegram/internal/action/TelegramActions.java

```diff
@@ -530,6 +537,104 @@ private boolean sendTelegramGeneral(@Nullable Long chatId, @Nullable String mess
         return sendTelegramPhoto(photoURL, caption, null, null);
     }
 
+    @RuleAction(label = "send a media group", description = "Send a Telegram media group using the Telegram API.")
+    public @ActionOutput(label = "Success", type = "java.lang.Boolean") boolean sendTelegramMediaGroup(
+            @ActionInput(name = "chatId") @Nullable Long chatId,
+            @ActionInput(name = "mediaUrls") @Nullable List<String> mediaUrls,
+            @ActionInput(name = "mediaTypes") @Nullable List<String> mediaTypes) {
+        return sendTelegramMediaGroup(chatId, mediaUrls, mediaTypes, null, null, null);
+    }
+
+    @RuleAction(label = "send a media group", description = "Send a Telegram media group using the Telegram API.")
+    public @ActionOutput(label = "Success", type = "java.lang.Boolean") boolean sendTelegramMediaGroup(
+            @ActionInput(name = "mediaUrls") @Nullable List<String> mediaUrls,
+            @ActionInput(name = "mediaTypes") @Nullable List<String> mediaTypes) {
+        return sendTelegramMediaGroup(null, mediaUrls, mediaTypes, null, null, null);
+    }
+
+    @RuleAction(label = "send a media group", description = "Send a Telegram media group using the Telegram API.")
+    public @ActionOutput(label = "Success", type = "java.lang.Boolean") boolean sendTelegramMediaGroup(
```

Comment:
Please add the missing static methods (required by the DSL rule engine).

Reply:
Yes, added.